packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -45,11 +45,11 @@ export type MafsGraphProps = {
     labels: InteractiveGraphProps["labels"];
     state: InteractiveGraphState;
     dispatch: React.Dispatch<InteractiveGraphAction>;
-    hintMode: boolean;
+    static: boolean | null | undefined;
```

Comment:
I'm not sure if types from other uses of `static` force this, but can we make this `static: boolean | undefined`?  I think there are many places that use this `X | null | undefined` pattern, but in reality, we rarely distinguish between `null` and `undefined`. 

This is a nitpicky thing, but it's another place where the type is broader/more complicated for no reason.

Reply:
The type `boolean | null | undefined` comes from [WidgetProps](https://github.com/Khan/perseus/blob/81db6a703677a3447da04ca05ee1707b7f97a851/packages/perseus/src/types.ts#L582), so I had to use the same type here.

I don't really feel comfortable changing WidgetProps, since it's so widely used (and it's one of our few defenses against writing widgets that can't handle production data).